core/src/main/scala/io/finch/Input.scala

```diff
@@ -67,29 +67,14 @@ final case class Input(request: Request, route: List[String]) {
    */
   def withForm(params: (String, String)*): Input = {
     // TODO: Figure out way to do that w/o Netty.
-    val dataFactory = new DefaultHttpDataFactory(false) // we don't use disk
-    val encoder = new HttpPostRequestEncoder(
-      dataFactory,
-      new DefaultHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.POST, request.path),
-      false
-    )
-
-    params.foreach {
-      case (k, v) => encoder.addBodyAttribute(k, v)
-    }
-
-    val req = encoder.finalizeRequest()
-
-    val content = if (req.isChunked) {
-      Buf(
-        Iterator.continually(encoder.nextChunk())
-        .takeWhile(c => !c.isLast)
-        .map(c => ChannelBufferBuf.Owned(c.getContent))
-        .toVector
-      )
-    } else ChannelBufferBuf.Owned(req.getContent)
-
-    withBody[Application.WwwFormUrlencoded](content, StandardCharsets.UTF_8)
+     val postRequest: Request = RequestBuilder()
+       .setHeader(Fields.UserAgent, "MyBot")
```

Comment:
Do we need these two headers? I'd imagine these makes little to no difference for local testing.

Reply:
@vkostyukov You're right. I removed them.